Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -876,7 +877,8 @@ function Set-IdentityEnvironmentAzureVariables {
         # Set Azure Settings
         $environmentVariables.Add("AzureActiveDirectorySettings__Authority", "https://login.microsoftonline.com/common")
         $environmentVariables.Add("AzureActiveDirectorySettings__DisplayName", "Azure AD")
-        $environmentVariables.Add("AzureActiveDirectorySettings__ClaimsIssuer", "https://login.microsoftonline.com/" + $claimsIssuer)
+        $environmentVariables.Add("AzureActiveDirectorySettings__ClaimsIssuer", "https://login.microsoftonline.com/" + $claimsIssuer.name)
```

Comment:
I noticed $claimsIssuer changed, but i dont see those changes, i take it the web service just parses differently.  Is that the case?

Reply:
The way it's grabbed from the install.config is slightly different now.  It used to be a string, but was changed to be a hashtable to get the alias to come along with it as well.